src/main/java/se/sundsvall/smloader/service/scheduler/CaseProcessingScheduler.java

```diff
@@ -0,0 +1,44 @@
+package se.sundsvall.smloader.service.scheduler;
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import se.sundsvall.dept44.requestid.RequestId;
+import se.sundsvall.smloader.service.OpenEService;
+import se.sundsvall.smloader.service.SupportManagementService;
+
+import java.time.LocalDateTime;
+
+@Service
+public class CaseProcessingScheduler {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(CaseProcessingScheduler.class);
+	private static final String LOG_IMPORT_STARTED = "Beginning of importing cases";
+	private static final String LOG_IMPORT_ENDED = "Import of cases has ended";
+	private static final String LOG_EXPORT_STARTED = "Beginning of exporting cases";
+	private static final String LOG_EXPORT_ENDED = "Export of cases has ended";
+
+	private final OpenEService openEService;
+	private final SupportManagementService supportManagementService;
+
+	public CaseProcessingScheduler(final OpenEService openEService, final SupportManagementService supportManagementService) {
+		this.openEService = openEService;
+		this.supportManagementService = supportManagementService;
+	}
+
+	@Scheduled(cron = "${scheduler.caseprocessing.cron.expression}")
+	@SchedulerLock(name = "processcases", lockAtMostFor = "${scheduler.shedlock-lock-at-most-for}")
+	public void execute() {
+		RequestId.init();
+
+		LOGGER.info(LOG_IMPORT_STARTED);
+		openEService.fetchAndSaveNewOpenECases(LocalDateTime.now().minusDays(1).withHour(0).withMinute(0).withSecond(0).withNano(0), LocalDateTime.now());
```

Comment:
Fetches one day back, each day. What happens if job fails one day (service broken or something). Will those cases created that day never be fetched?

Reply:
This handling is not ready yet and this is for testing purpose only. Will be managed by config when it's ready.

Reply:
Added a config-attribute for days to fetch now.